Docker/OutputFilter/OutputFilter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Keboola\DockerBundle\Docker\OutputFilter;
+
+class OutputFilter implements OutputFilterInterface
+{
+    const REPLACEMENT = '*****';
```

Comment:
nedáme tam něco jako `[hidden]`? na travisu je `[secure]`

Reply:
jo, urcite spis teda `[hiden]`, secure zni, ze je to snad jako secure, coz uplne neni :)